### PR TITLE
font awesome upgrade across the board

### PR DESCRIPTION
**Pull request by caijim**
@caesar2164  some todo items (just ones involving fa-stack-base)

### CODE REVIEW

File: lms/static/coffee/src/instructor_dashboard/data_download.coffee [hunk 1]
```diff
@@ -201,11 +201,11 @@ class ReportDownloads
         formatter: (row, cell, value, columnDef, dataContext) ->
           data_link ='<a class="course-forums-data" href="' + dataContext['url'] + '">' + dataContext['name'] + '</a>'
           if dataContext['name'].indexOf("course_forums") > -1
-            graph_button = _.template('<a class="graph-forums"><i class="icon-bar-chart"></i> <%= label %></a>',
+            graph_button = _.template('<a class="graph-forums"><i class="fa-bar-chart-o"></i> <%= label %></a>',
```
Comment (caesar2164) [inline lms/static/coffee/src/instructor_dashboard/data_download.coffee]: let's make this just the regular `fa-bar-chart` icon

File: cms/templates/widgets/utility-captions-components.html [hunk 2]
```diff
@@ -14,23 +14,26 @@
...
+        <span class="fa-stack sync">
+            <!-- Giulio to fix icon-stack-base -->
+          <i class="fa-square icon-stack-base"></i>
+          <i class="fa-link fa-inverse fa-lg sync"></i>
         </span>
       </div>
 
       <div class="item-actions">
-        <i class="icon-spinner icon-spin icon-large"></i>
+        <i class="fa-spinner fa-spin fa-lg"></i>
```
Comment (caesar2164) [inline cms/templates/widgets/utility-captions-components.html]: lets make this `fa-spin` `fa-pulse` as well

**Comment by caesar2164** [comment]
@caijim - there you go added a commit with my changes...

**Comment by caijim** [comment]
@caesar2164 looks great to me. shall i squash and rebase?

**Comment by caesar2164** [comment]
Yep @caijim, squash away!

**Comment by caesar2164** [comment]
btw, I added class "fa" to all the icons, cause that's what the new font-awesome version wants...

**Comment by caijim** [comment]
yep, thanks for catching that :)

**Comment by stvstnfrd** [comment]
Looks good @caijim .

I'll go ahead and pull this into my branch later today. I'll close this PR if I cherry-pick instead of merging, to signify completion.

**Comment by stvstnfrd** [comment]
I cherry-picked this at 468aede756aed515e8365171d444ca9c1112ca64

File: cms/templates/course_outline.html [hunk 3]
```diff
@@ -48,8 +48,7 @@ <h2 class="title title-3">${_("This course was created as a re-run. Some manual
         </li>
       </ul>
     </div>
-  </div>
-  %endif
+  </div>  %endif
```
Comment (stvstnfrd) [inline cms/templates/course_outline.html]: CC: @caijim 

I didn't notice it at the time, but this turns out to be a syntax error;

`SyntaxException: Unterminated control keyword: 'if' in file '/edx/app/edxapp/edx-platform/cms/templates/course_outline.html' at line: 31 char: 1`

Comment (stvstnfrd) [inline cms/templates/course_outline.html]: I've already fixed it locally, just reverted the change.

Comment (caijim) [inline cms/templates/course_outline.html]: ah, my bad for this erroneous line

Comment (caijim) [inline cms/templates/course_outline.html]: do you want me to submit another PR to fix this?

Comment (stvstnfrd) [inline cms/templates/course_outline.html]: Already taken care of :)

Thanks though!
